### DISCUSSION TITLE
vanilla Scoreboard data

### DISCUSSION THREAD

**Discussion by Lobskii** [ShaneBeee/SkBee#490] [Q&A]
Can SkBee get and set values from vanilla minecraft scoreboard objectives?
I have seen documentation examples on SkUnity that seem to say SkBee can do this, but the code gives errors when I try to use it.

Example documentation from SkUnity:
https://docs.skunity.com/syntax/search/id:10194
Scoreboard - Objective Score
score of %entities/strings% for %objective%
Get/Set the score of an entity/string for an objective.

**Comment by ShaneBeee** [comment]
yes, SkBee can do scoreboard stuff.
I recommend using the SkriptHub docs, as skUnity docs have been having issues.

if you require any further help, please join skUnity's discord, and im sure someone there can help you 👍

**Comment by Lobskii** [comment]
I have tried using the same syntax as the example on SkriptHub but it still does not seem to work.
I suspected maybe I was supposed to use the Objective Get example to set the objective from the vanilla minecraft objectives, I dont get errors from it when loading the skript, but it does not set the variable I am using.
Example Code:
command ffpoints [<player>]:
    permission: op
    trigger:
        set {_obj} to objective with id "ff_GlobalPoints"
        set {_ff_currpoints} to score of player for {_obj}
        send "%player% has a points value of %_ff_currpoints%"

error when loading: 
Can't understand this expression: _ff_currpoints

Which says to me it cant set the _ff_currpoints variable?

I did post some thing in SkUnity's discord asking about it before I found some thing saying SkBee could interact with scoreboard objectives.

**Comment by ShaneBeee** [comment]
yeah, if you need help you're going to have to ask for help on discord.
I don't provide support on the issue tracker, just meant for issues and suggestions
